<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-07-13T07:50:15Z" name="Suite" finished-at="2023-07-13T07:50:37Z" duration-ms="21815">
    <groups>
    </groups>
    <test started-at="2023-07-13T07:50:15Z" name="Test" finished-at="2023-07-13T07:50:37Z" duration-ms="21815">
      <class name="com.automationstore.testScript.Home">
        <test-method signature="testnavigateToLogin()[pri:0, instance:com.automationstore.testScript.Home@6973bf95]" started-at="2023-07-13T13:20:15Z" name="testnavigateToLogin" finished-at="2023-07-13T13:20:37Z" duration-ms="21718" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testnavigateToLogin -->
      </class> <!-- com.automationstore.testScript.Home -->
      <class name="com.automationstore.testScript.Login">
        <test-method signature="testnavigateToLogin()[pri:0, instance:null]" started-at="2023-07-13T13:20:37Z" name="testnavigateToLogin" finished-at="2023-07-13T13:20:37Z" duration-ms="15" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Can't invoke public void com.automationstore.testScript.Home.testnavigateToLogin() throws java.lang.InterruptedException: either make it static or add a no-args constructor to your class]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Can't invoke public void com.automationstore.testScript.Home.testnavigateToLogin() throws java.lang.InterruptedException: either make it static or add a no-args constructor to your class
	at org.testng.internal.Utils.checkInstanceOrStatic(Utils.java:790)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:42)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testnavigateToLogin -->
        <test-method signature="testLogin()[pri:0, instance:null]" started-at="2023-07-13T13:20:37Z" name="testLogin" finished-at="2023-07-13T13:20:37Z" duration-ms="4" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Can't invoke public void com.automationstore.testScript.Login.testLogin() throws java.lang.InterruptedException: either make it static or add a no-args constructor to your class]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Can't invoke public void com.automationstore.testScript.Login.testLogin() throws java.lang.InterruptedException: either make it static or add a no-args constructor to your class
	at org.testng.internal.Utils.checkInstanceOrStatic(Utils.java:790)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:42)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
      </class> <!-- com.automationstore.testScript.Login -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
